payload/dropper/jjs.go

```diff
@@ -0,0 +1,48 @@
+package dropper
+
+import (
+	"encoding/base64"
+	"fmt"
+	"slices"
+
+	"github.com/vulncheck-oss/go-exploit/output"
+)
+
+// Generates a script that can be used to create a file on disk via jjs (Java javascript). And if
+// execute is true, then toggle file permissions and execute the payload. The number of chunks are
+// in bytes.
+func (jjs *JJSScriptPayload) WriteChunks(content []byte, targetFile string, chunkSize int, execute bool) []string {
```

Comment:
Looking at the function signature, I believe we could have a unite test for this?